ansible/common/php-fpm/tasks/main.yml

```diff
@@ -19,9 +19,6 @@
       - php{{ php.version }}-imagick
       - php{{ php.version }}-imap
       - php{{ php.version }}-mbstring
-      - '{{ "php" + php.version + "-mcrypt"
-              if (php.version is version_compare("7.2","<"))
-              else [] }}'
```

Comment:
I was moderately fond of this conditional 😢

Reply:
Yeah I respected it, but it was broken once switched to the array format, don't know why.